CoreScriptsRoot/Modules/Settings/Pages/ReportAbuseMenu.lua

```diff
@@ -165,6 +165,45 @@ local function Initialize()
 			submitText.ZIndex = 1
 		end
 
+		local function fieldsCompleted()
```

Comment:
we should rename this to `areFieldsCompleted()` so that the function has a verb name.